src/views/Question.vue

```diff
@@ -200,6 +212,10 @@ watch:{
   pointer-events: none;
 }
 
+.router-disabled {
+  pointer-events: none;
+}
+
```

Comment:
Instead of applying this class to the `button`, there is an alternative way where [`<button>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) also accepts the `disabled` property.